tests/data/cases/stub_empty_line.py

```diff
@@ -0,0 +1,7 @@
+class C:
+    def f(self):
+        
```

Comment:
Maybe add test cases (in the same file) where there are comments in various places around the `...`, e.g. on a line by itself or right after `:`.